bedrock/firefox/templates/firefox/more.html

```diff
@@ -259,13 +230,30 @@ <h2>{{ ftl('learn-more-about-firefox', fallback='firefox-products-are') }}</h2>
         desc=ftl('firefox-more-translate-more-than'),
         link_url=url('firefox.features.translate')
       )}}
+    </div>
+    <div class="mzp-l-card-third">
+      {{ card(
+        title=ftl('firefox-more-protection'),
+        ga_title='Features Private Browsing',
+        image=resp_img('img/firefox/more/private-browsing.jpg', optional_attributes={"loading": "lazy", "class": "mzp-c-card-image"}),
+        desc=ftl('were-obsessed-with'),
+        link_url=url('firefox.features.private-browsing')
+      )}}
+
+      {{ card(
+        title=ftl('firefox-more-avoid-misinformation-heading'),
+        ga_title='Avoid Misinformation',
+        image=resp_img('img/firefox/more/avoid-misinformation.jpg', optional_attributes={"loading": "lazy", "class": "mzp-c-card-image"}),
+        desc=ftl('firefox-more-avoid-misinformation-desc'),
+        link_url=url('firefox.more.misinformation')
+      )}}
 
       {{ card(
         title=ftl('firefox-more-a-guide-to'),
         ga_title='Safe Passwords',
-        desc=ftl('firefox-more-more-and-more'),
         image=resp_img('img/firefox/more/passwords.png', optional_attributes={"loading": "lazy", "class": "mzp-c-card-image"}),
-        link_url=url('firefox.privacy.passwords')
+        desc=ftl('firefox-more-more-and-more'),
+        link_url='https://support.mozilla.org/kb/create-secure-passwords-keep-your-identity-safe'
```

Comment:
**FYI:** this card stays, even though it doesn't add any backlink value now that it points externally… but it helps to keep the layout ok~ish — with it also removed it'd be tricky to align the cards to look somewhat good.

It can be removed eventually, when there's better/new content to be placed in the `more` page.

Reply:
OK I didn't really like one row where the paragraph length differences in quarters made it quite unpleasant to read — and actually managed to move the cards around to balance out the line count for each row a bit better and change some quarters to thirds — therefore also being able to get rid of this last card. Poof!💨

Reply:
🪄